### DISCUSSION TITLE
Command to revert `brew create`

### DISCUSSION THREAD

**Discussion by abitrolly** [Homebrew/discussions#2017] [Writing Formulae/Casks] [answered]
I've used a wrong URL for `brew create`. Is there a command to remove the new definition or overwrite it? `brew remove` didn't work, because it only operates on installed formulas. `brew create --force` didn't work either, complaining that `.rb` file already exists.

**Comment by carlocab** [comment, marked as answer]
```
rm "$(brew --repository homebrew/core)/Formula/formula-you-want-to-remove.rb"
```
or
```
git -C "$(brew --repository homebrew/core)" clean -f # assumes you're happy to delete any other untracked files
```

**Reply by abitrolly** [reply]
I guess it is not used very often, so there is no `brew <command>` for that.
